.github/workflows/assets-sync.yml

```diff
@@ -14,55 +14,41 @@ jobs:
         with:
           repos: |
             pion/datachannel
+            pion/dcnet
             pion/dtls
             pion/ice
             pion/interceptor
             pion/logging
             pion/mdns
+            pion/mediadevices
             pion/opus
             pion/peerconnection_explainer
+            pion/portmap
             pion/randutil
             pion/rtcp
             pion/rtp
+            pion/rtpio
             pion/sctp
             pion/sdp
             pion/srtp
+            pion/template
             pion/transport
             pion/turn
             pion/udp
             pion/webrtc
-          root_dir: ci
-          rm: |
-            .github/hooks/*
-            .github/workflows/renovate-go-mod-fix.yaml
-            .github/generate-authors.sh
-            .github/lint-*.sh
-          push_interval: 10
-          git_user: Pion Bot
-          git_email: 59523206+pionbot@users.noreply.github.com
-          github_token: ${{ secrets.PIONBOT_ASSETS_SYNC_TOKEN }}
-          commit_message: |
-            Update CI configs to %v
-
-            Update lint scripts and CI configs.
-      - name: sync CI configs
-        uses: at-wat/assets-sync-action@v0
-        with:
-          repos: |
             pion/stun
           root_dir: ci
           rm: |
+            codecov.yml
```

Comment:
This will remove codecov.yml from every repository. Is it intended?

Reply:
Thanks. That was a mistake. I removed it